test/eme.test.js

```diff
@@ -494,6 +494,13 @@ QUnit.test('5 July 2016 lifecycle', function(assert) {
 });
 
 QUnit.test('getSupportedKeySystem error', function(assert) {
+
+  // Skip this test in Safari, getSupportedKeySystem is never used in Safari.
+  if (videojs.browser.IS_ANY_SAFARI) {
```

Comment:
It may be better to wrap this test in this check, so that we don't have a false positive for this test on safari.

Reply:
Reasonable! 👍